src/lm_buddy/integrations/wandb/artifact_utils.py

```diff
@@ -91,57 +91,21 @@ def build_directory_artifact(
 def build_table_artifact(
     artifact_name: str,
     artifact_type: ArtifactType,
-    columns: list[str],
-    tables: dict[str, list[list[Any]]],
+    tables: dict[str, pd.DataFrame],
 ) -> wandb.Artifact:
     """Build an artifact containing one or more table entries.
 
     Args:
         artifact_name (str): Name of the artifact.
         artifact_type (ArtifactType): Type of artifact.
-        columns (list[str]): Column names for the tables.
-        tables (dict[str, list[list[Any]]]): Mapping from table name to table rows.
+        tables (dict[str, pd.DataFrame]): Mapping from table name to table data
```

Comment:
did pandas end up being easier than the previous implementation?

Reply:
it did, yeah. We really just need a data structure for a generic tabular data. I was storing raw lists before, but `pd.DataFrame` was a bit easier to work with and it integrates automatically w/ W&B Tables.